airflow/executors/kubernetes_executor.py

```diff
@@ -276,13 +289,14 @@ def __init__(self):
     # and only return a blank string if contexts are not set.
     def _get_security_context_val(self, scontext):
         val = configuration.get(self.kubernetes_section, scontext)
-        if len(val) == 0:
-            return val
+        if not val:
+            return 0
```

Comment:
Don't you change the behaviour like this?

Shouldn't it be
```suggestion
            return []
```
or just `return val` ?

Reply:
Good catch! But it should return "" not [] (see the comment above the method :).

Reply:
Resolved!